src/status_im/contexts/wallet/send/save_address/view.cljs

```diff
@@ -0,0 +1,97 @@
+(ns status-im.contexts.wallet.send.save-address.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.theme]
+    [react-native.core :as rn]
+    [status-im.common.floating-button-page.view :as floating-button-page]
+    [status-im.contexts.wallet.send.save-address.style :as style]
+    [status-im.contexts.wallet.sheets.network-preferences.view
+     :as network-preferences]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- address-view
+  [address]
+  (let [network-details (rf/sub [:wallet/network-preference-details])]
+    [rn/view {:style style/address-container}
+     [quo/data-item
+      {:status          :default
+       :size            :default
+       :subtitle-type   :default
+       :label           :none
+       :blur?           false
+       :icon-right?     true
+       :right-icon      :i/advanced
+       :card?           true
+       :title           (i18n/label :t/address)
+       :custom-subtitle (rn/use-callback
+                         (fn [] [quo/address-text
+                                 {:networks network-details
+                                  :address  address
+                                  :format   :long}]))
+       :on-press        (rn/use-callback
+                         (fn []
+                           (rf/dispatch [:show-bottom-sheet
+                                         {:content
+                                          (fn []
+                                            [network-preferences/view
+                                             {:on-save (fn [])}])}])))
+       :container-style style/data-item}]]))
+
+(defn view
+  []
+  (let [[address-label set-address-label] (rn/use-state "")
+        [address-color set-address-color] (rn/use-state :blue)
+        placeholder                       (i18n/label :t/address-name)]
+     [floating-button-page/view
+      {:footer-container-padding 0
+       :header                   [quo/page-nav
+                                  {:type                :no-title
+                                   :background          :blur
+                                   :icon-name           :i/close
+                                   :on-press            (rn/use-callback #(rf/dispatch [:navigate-back]))
+                                   :accessibility-label :save-address-top-bar}]
+       :footer                   [quo/button
+                                  {:accessibility-label :save-address-button
+                                   :type                :primary
+                                   :container-style     style/save-address-button}
+                                  (i18n/label :t/save-address)]
+
+       :customization-color      address-color
+       :gradient-cover?          true}
+
```

Comment:
why empty lines? between elements

Reply:
That's how I was doing it in my last project. I have removed the empty lines to stay consistent with this codebase.